multi-arch-builders/coreos-s390x-rhcos-builder.bu

```diff
@@ -62,6 +62,20 @@ storage:
             if ! $(lszdev | grep -q "${DISK_LUN}"); then
                 echo "Adding LUN to system"
                 echo "${DISK_LUN}" > /sys/bus/ccw/drivers/zfcp/${DISK_FCP}/${DISK_WWWN}/unit_add
+                echo "Waiting for disk to show up"
```

Comment:
I feel like we should be able to use systemd unit primitives to wait for this device to exist before running the `secex-data-volume.service` service

Reply:
We could split the service into 2 parts, the first one adding the device, the 2nd one mounting it. However I don't really think that makes sense.
The main issue with this is that the LPAR does not scan for "new" LUNs automatically, since I do not want to install FCOS to the wrong disk.

Reply:
Oh I see. The `echo "${DISK_LUN}"` is what causes the device to be added to the system?

Reply:
Is there a way to do that with a configuration file (static) versus an `echo` at runtime (dynamic)?

Reply:
Either way I'll resolve this conversation.

Reply:
The official docs for RHEL are: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-post-installation-fcp-attached-luns-s390